src/mscorlib/src/System/Runtime/Intrinsics/Vector128DebugView.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics
+{
+    public struct Vector128DebugView<T> where T : struct
```

Comment:
This should be `internal class`. (To match what all other debug views look like.)

Reply:
Oops. Changed this when I was testing the debug marshalling issue, forgot to revert.

Reply:
Fixed.